pkg/ExternalPackages/versions.props

```diff
@@ -2,19 +2,28 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
   <ItemGroup>
-    <ExternalPackage Include="runtime.win7-amd64.runtime.native.System.Data.SqlClient.sni">
+    <ExternalPackage Include="runtime.win7-x64.runtime.native.System.Data.SqlClient.sni">
```

Comment:
I see.  We really need to consolidate this state.  I have some suggestions below.  The way I built packaging targets was to have a data-driven process for calculating versions.  That's how we already handle stable vs prerelease.  This is just an added feature there where we need to lookup pre-release versions.